suite-native/accounts/src/utils.ts

```diff
@@ -76,3 +77,21 @@ export const groupAccountsByNetworkAccountType = A.groupBy((account: Account) =>
 
     return `${networkName} ${formattedAccountType} accounts`;
 });
+
+export const sortAccountsByNetworksAndAccountTypes = (accounts: readonly Account[]) => {
```

Comment:
Could you please create some simple unit test for this function? 🙏

Reply:
Good point. Adding it...